apps/condo/domains/user/access/SupportMessagesForwardingService.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Generated by `createservice user.SupportMessagesForwardingService`
+ */
+const { throwAuthenticationError } = require('@condo/domains/common/utils/apolloErrorFormatter')
+
+async function canSupportMessagesForwarding ({ authentication: { item: user } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+    if (user.isAdmin) return true
+
+    return { user: { id: user.id } }
```

Comment:
you can use only true/false for custom mutations

Reply:
Fixed.
`return true`